ui/src/message_popup/popup_question.jsx

```diff
@@ -155,8 +192,20 @@ export default React.createClass({
                 examples={examples}/>  
             </div>}
         </VelocityTransitionGroup>
+        <Snackbar
+          open={seconds >= 90}
+          message="Remember, these are supposed to be quick responses."
+          autoHideDuration={10000000}
```

Comment:
Nice, this seems like a good way to provide this!  What's the large `autoHideDuration` for here?